pages/getting-started/cloud-shell-quickstart.md

```diff
@@ -24,11 +24,76 @@ Navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Clo
 
 ### Set up a Cloud Provider
 
-Choose the option to set up your Cloud Provider in the browser. Plural can be set up on AWS, GCP, or Azure.
+Choose the option to continue your setup in the browser. If you'd prefer to use the CLI, choose "Local terminal" and switch to the CLI quickstart.
```

Comment:
Probably "Choose an option"?

Reply:
i'm assuming they're trying to set up in-browser, which is why it's directive